jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -2210,6 +2210,7 @@ void java_bytecode_convert_methodt::convert_invoke(
                          id2string(symbol.base_name) + "()";
     symbol.type = arg0.type();
     symbol.type.set(ID_access, ID_private);
+    to_java_method_type(symbol.type).set_is_final(true);
```

Comment:
A comment that repeats the commit message could be added for this (e.g. just above, together with the comments about accessibility).

Reply:
Makes sense. Added comment in function header.